src/model/backup.ts

```diff
@@ -144,7 +150,9 @@ export function findLatestBackUp(
 export async function restoreCloudBackup(
   password: BackupPassword,
   userData: BackupUserData | null,
-  backupSelected: string | null
+  backupSelected: string | null,
+  doItIfStartPINCreation = empty,
+  doItIfFinishPINCreation = empty
```

Comment:
Those names sound very weird to me. Can it be something like `onBeforePINCreated` and `onAfterPINCreated`?

Reply:
sure